.github/workflows/release.yaml

```
@@ -30,14 +25,19 @@ jobs:
\ No newline at end of file
+          POSTHOG_HOST: ${{ secrets.POSTHOG_HOST }}
+
+      - name: Generate Changelog
+        run: bun run release
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+
+      - name: Publish CLI to NPM
+        run: cd apps/cli && bun publish --access public
+        env:
+          NPM_CONFIG_TOKEN: ${{ secrets.NPM_TOKEN }}
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)